plugins/module_utils/constants.py

```diff
@@ -88,3 +88,5 @@
 MATCH_SMU_OPERATION_MAPPING = dict(smu_install="smuInstall", smu_uninstall="smuUninstall")
 
 MATCH_SMU_OPERATION_FLAGS_MAPPING = dict(smu_reload_immediate="smuReloadImmediate", smu_reload_skip="smuReloadSkip")
+
+MATCH_ACTION_RULE_SET_METRIC_TYPE_MAPPING = {"ospf_type_1":"ospf-type1", "ospf_type_2":"ospf-type2", "":""}
```

Comment:
The empty string can be removed since the only input parameters possible are "ospf_type_1", "ospf_type_2" or None

```suggestion
MATCH_ACTION_RULE_SET_METRIC_TYPE_MAPPING = {"ospf_type_1":"ospf-type1", "ospf_type_2":"ospf-type2"}
```

If the empty string is needed a default in the `get(value, "")` can be specified